### ISSUE TITLE
Docs fail to build

### ISSUE THREAD

**Issue by cswinter** [entity-neural-network/enn-trainer#29]
Docs are failing to build: https://readthedocs.org/projects/enn-trainer/builds/20741572/

```
Looking in links: https://data.pyg.org/whl/torch-1.12.0+cpu.html
Collecting sphinx==4.5.0 (from -r docs/requirements.txt (line 1))
  Downloading Sphinx-4.5.0-py3-none-any.whl (3.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 158.9 MB/s eta 0:00:00
Collecting sphinx_rtd_theme==1.0.0 (from -r docs/requirements.txt (line 2))
  Downloading sphinx_rtd_theme-1.0.0-py2.py3-none-any.whl (2.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.8/2.8 MB 199.7 MB/s eta 0:00:00
Collecting autoapi==2.0.1 (from -r docs/requirements.txt (line 3))
  Downloading autoapi-2.0.1.tar.gz (11 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting torch==1.12.0 (from -r docs/requirements.txt (line 4))
  Downloading torch-1.12.0-cp39-cp39-manylinux1_x86_64.whl (776.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 776.3/776.3 MB 199.4 MB/s eta 0:00:00
Collecting torch-scatter (from -r docs/requirements.txt (line 6))
  Downloading torch_scatter-2.1.1.tar.gz (107 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.6/107.6 kB 228.5 MB/s eta 0:00:00
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-kbsqd03a/torch-scatter_4ad4ce6fdc904714aab2bbfaf80b90d7/setup.py", line 8, in <module>
          import torch
      ModuleNotFoundError: No module named 'torch'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by cswinter** [comment]
First failing build: https://readthedocs.org/projects/enn-trainer/builds/20315541/
Last successful build: https://readthedocs.org/projects/enn-trainer/builds/19778172/

Issue seems to be related to torch-scatter setup.py failing on torch import. Maybe need different version of torch-scatter.

**Comment by cswinter** [comment]
Partial progress with https://github.com/entity-neural-network/enn-trainer/pull/30 running into new error now: https://readthedocs.org/projects/enn-trainer/builds/23477576/

```
 Running Sphinx v4.5.0
loading translations [en]... done

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/registry.py", line 438, in load_extension
    metadata = setup(app)
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinxcontrib/applehelp/__init__.py", line 230, in setup
    app.require_sphinx('5.0')
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/application.py", line 393, in require_sphinx
    raise VersionRequirementError(version)
sphinx.errors.VersionRequirementError: 5.0

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/cmd/build.py", line 272, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/application.py", line 219, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/application.py", line 380, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/enn-trainer/envs/latest/lib/python3.9/site-packages/sphinx/registry.py", line 441, in load_extension
    raise VersionRequirementError(
sphinx.errors.VersionRequirementError: The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.

Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version. 
```

**Comment by cswinter** [comment]
Remaining issue fixed by #31
